app/routes/profile.js

```diff
@@ -6,5 +6,9 @@ const { Route } = Ember;
 export default Route.extend(AuthenticatedRouteMixin, {
   titleToken() {
     return this.l10n.t('Profile');
+  },
+
+  model() {
+    return this.get('authManager').currentUserModel;
```

Comment:
@geekyd you should _always_ use `get` with `dot notation` for all ember objects. 

```js
this.get('authManager. currentUser')
```